vertx-grpc-common/src/main/java/io/vertx/grpc/common/impl/ServiceNameImpl.java

```diff
@@ -61,4 +63,17 @@ public String pathOf(String method) {
       return '/' + packageName + '.' + name + '/' + method;
     }
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
can you add curly brace in statements ?

Reply:
I hope the changes I made follow the code-style you desire.